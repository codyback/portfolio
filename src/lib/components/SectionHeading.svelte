<script lang="ts">
  export let heading: string;
</script>

<div class="border-b-2 border-black dark:border-neutral-500 pb-3">
  <h3 class="uppercase text-2xl font-bold text-right">{heading}</h3>
</div>
<div class="border-t-4 border-black dark:border-neutral-500 mt-0.5" />
