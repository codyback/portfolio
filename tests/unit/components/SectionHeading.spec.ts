import SectionHeading from '@/lib/components/SectionHeading.svelte';
import { render } from '@testing-library/svelte';
import { describe, it, expect } from 'vitest';

describe('Sidebar', () => {
	it('renders correctly', () => {
		const { container } = render(SectionHeading, { heading: 'Testing' });

		expect(container).toMatchSnapshot();
	});
});
