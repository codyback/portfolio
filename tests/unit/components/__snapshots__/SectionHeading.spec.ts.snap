// Vitest Snapshot v1

exports[`Sidebar > renders correctly 1`] = `
<body>
  <div>
    <div
      class="border-b-2 border-black dark:border-neutral-500 pb-3"
    >
      <h3
        class="uppercase text-2xl font-bold text-right"
      >
        Testing
      </h3>
    </div>
     
    <div
      class="border-t-4 border-black dark:border-neutral-500 mt-0.5"
    />
    <!--&lt;SectionHeading&gt;-->
  </div>
</body>
`;
